Release.toml

```diff
@@ -152,3 +152,4 @@ version = "1.10.0"
     "migrate_v1.10.0_aws-control-container-v0-6-3.lz4",
     "migrate_v1.10.0_public-control-container-v0-6-3.lz4"
 ]
+"(1.10.0, 1.10.1)" = []
```

Comment:
This needs to include migrations for `settings.container-runtime` added in: https://github.com/bottlerocket-os/bottlerocket/pull/2494/files

Reply:
Oh My bad! thanks